### PR TITLE
picom-next: init at unstable-2021-10-31

### PR DESCRIPTION
**Pull request by GKasparov**
###### Motivation for this change
Picom master branch does not support dual_kawase blur and the latest release is quite old. This will provide an updated picom with next as the updates (still pinned to a commit) and will provide new features such as dual_kawase and other patches 

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -4201,6 +4201,12 @@
     githubId = 1713676;
     name = "Luis G. Torres";
   };
+  GKasparov = {
+    email = "mizozahr@gmail.com";
+    github = "GKasparov";
+    githubId = 60962839;
```
Comment (thiagokokada) [inline maintainers/maintainer-list.nix:4207]: Checks:

```
$ curl -Ssq https://api.github.com/users/GKasparov | jq -r .id
60962839
```

File: pkgs/applications/window-managers/picom/picom-next.nix [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{ picom, lib, fetchFromGitHub }:
+
+picom.overrideAttrs (oldAttrs: rec {
+  pname = "picom-next";
+  version = "unstable-2021-10-31";
+  src = fetchFromGitHub {
+    owner = "yshui";
+    repo = "picom";
+    rev = "fade045eadf171d2c732820d6ebde7d1943a1397";
```
Comment (thiagokokada) [inline pkgs/applications/window-managers/picom/picom-next.nix:9]: Checks: https://github.com/yshui/picom/commit/fade045eadf171d2c732820d6ebde7d1943a1397

**Review by thiagokokada — APPROVED** [review]
- Code wise :ok: 
- ~Commits :ok:~ (see below)

Result of `nixpkgs-review pr 144696` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>picom-next</li>
  </ul>
</details>

**Comment by thiagokokada** [comment]
https://github.com/NixOS/nixpkgs/pull/144696/commits/4b5557d9ae09f5a70388f94d5e8dabc99b81b78a

Why `picom-tryone` :thinking: ?

**Comment by GKasparov** [comment]
> [4b5557d](https://github.com/NixOS/nixpkgs/commit/4b5557d9ae09f5a70388f94d5e8dabc99b81b78a)
> 
> Why `picom-tryone` thinking ?

Ugh did I get the name wrong

**Comment by GKasparov** [comment]
> [4b5557d](https://github.com/NixOS/nixpkgs/commit/4b5557d9ae09f5a70388f94d5e8dabc99b81b78a)
> 
> Why `picom-tryone` instead of `picom-next` thinking ?

Renamed the commit

**Review by thiagokokada — APPROVED** [review]

**Comment by thiagokokada** [comment]
@GKasparov Thanks for your contribution!

**Pull request merged**